### DISCUSSION TITLE
Is it possible to do range search with `Delaunay_triangulation_3` ?

### DISCUSSION THREAD

**Discussion by ImperatorS79** [CGAL/cgal#8450] [Q&A] [answered]
In 2D, `CGAL::range_search` from `CGAL/range_search_delaunay_2.h` can be used, but I couldn't find the equivalent for `Delaunay_triangulation_3`.

**Comment by mglisse** [comment, marked as answer]
I don't think there is a function that does it (I only see Delaunay_2 and Delaunay_d versions), but you could take the code for Delaunay_2 and adapt it to Delaunay_3, it is not very long and should work almost as is (replacing Circle_2 with Sphere_3, etc): insert the point and do a dfs (the geometric property allowing to do a dfs should be true in any dimension).

**Reply by ImperatorS79** [reply]
This what I ended up doing.
